src/main/java/org/alfresco/web/app/servlet/WebScriptSSOAuthenticationFilter.java

```diff
@@ -102,8 +102,8 @@ public void doFilter(ServletContext context, ServletRequest sreq, ServletRespons
         String pathInfo = requestURI.substring((req.getContextPath() + req.getServletPath()).length());
         
         if (getLogger().isDebugEnabled())
-            getLogger().debug("Processing request: " + requestURI + " SID:" +
-                    (req.getSession(false) != null ? req.getSession().getId() : null));
+        	// MNT-20201 (LM-190213): display current session id or new one if not exist
+        	getLogger().debug("Processing request: " + requestURI + " SID:" + req.getSession().getId());
```

Comment:
A new session should not be created during debug output. The previous code is correct.
This issue is a false positive one.